gr-blocks/lib/complex_to_interleaved_short_impl.h

```diff
@@ -19,10 +19,11 @@ namespace blocks {
 class BLOCKS_API complex_to_interleaved_short_impl : public complex_to_interleaved_short
 {
 private:
+    float d_scalar;
     bool d_vector;
 
 public:
-    complex_to_interleaved_short_impl(bool vector);
+    complex_to_interleaved_short_impl(float scale_factor, bool vector);
```

Comment:
Since the scale_factor currently is hardcoded to 1.0 - should the new parameter to these blocks be placed at the end with the default value so that the API is unchanged?  That way we can backport to 3.8 and also not force anyone to modify their flowgraphs on master.